components/nodemanager-service/pgdb/migration/sql/04_nodes_add_fields.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE IF EXISTS nodes
+  ADD COLUMN IF NOT EXISTS last_scan JSON not null default '{}'::json,
+  ADD COLUMN IF NOT EXISTS last_run JSON not null default '{}'::json;
```

Comment:
It looks from some places in the code that these two are exclusive, ie, a node should only have last_scan or last_run, but not both. Is that true? If so this could probably be simplified by not using json for this and just having the fields be regular columns on the nodes table.

Reply:
a node can have one, the other, or both - but a single report will only have one or the other

Reply:
^ given this, doing individual columns would mean 8 columns...vs 2 with json